### ISSUE TITLE
Name property or Aria-label is not provided for ComboBox

### ISSUE THREAD

**Issue by yangster23** [microsoft/fluentui#16981]
### Environment Information

- **Package version(s)**: 7.64
- **Browser and OS versions**: 
Windows Version: Win10 
Chrome Version: 87.0.4280.88 (Official Build) (64-bit)
### Describe the issue:

Accessible name property is not provided for combo box despite attempting to pass aria-label property. Accessibility users won't be able to understand purpose of the control. Using Accessibility Insight's fast pass, following error is provided:

![image](https://user-images.githubusercontent.com/5368099/107839274-c0052d80-6d78-11eb-940d-907aded3ef42.png)

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/jonny-yang/pen/bGBgwJV

#### Actual behavior:

The name property or aria-label attribute does not exist for combo box. 

![image](https://user-images.githubusercontent.com/5368099/107839424-bc25db00-6d79-11eb-9014-3f17fb4ddd0b.png)

As no name property assigned to combo box, screen readers like NVDA is only announcing "combo box  collapsed  has auto complete" if there is no data.

#### Expected behavior:

The name property or aria label should be provided for combo box.

### Documentation describing expected behavior

https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

**Comment by gouttierre** [comment]
@yangster23 - thank you for the feed back and repo steps.

@khmakoto - Would you be able to if this is a regression or if this behavior is an accessibility issue?
